modules/security-context-constraints-command-reference.adoc

```diff
@@ -119,8 +125,8 @@ $ oc edit scc <scc_name>
 
 [NOTE]
 ====
-To preserve customized SCCs during upgrades, do not edit settings on
-the default SCCs.
-//other than priority, users, groups, labels, and annotations.
+To allow access to use an SCC, do not update any of its fields. Instead, refer to
+xref:../authentication/managing-security-context-constraints.html#role-based-access-to-ssc_configuring-internal-oauth[Role-based access to Security Context Constraints]
```

Comment:
You can't have an xref in a module.

Reply:
I was wondering about that (even though it worked when I tried but I had no idea whether what I am doing is right or not).

How do you refer to other subsections, then?

Reply:
@kalexand-rh is there a way to refer to other subsections, then? The above seem to have worked, but I understand why it shouldn't be allowed from the logical perspective.

Reply:
@stlaz, it does render correctly, but it only does so in the context of this particular assembly. The modules need to be able to stand alone without links to any other content so that we can use any of them in any other context that's required. The "no xrefs in modules" ban is CCS-wide, and it's a ban that's resisted many, many arguments against them. I will be happy when it's lifted.

The best mitigation strategy is to add an xref from one assembly to another, but that won't work in this case. @huffmanca's on vacation, but I'm going to check with him about his original intent with this assembly and see what the best strategy is.

Reply:
@kalexand-rh thanks for the heads-up, did you and @huffmanca reach any conclusion about how to do this properly? Is it possible to do now?